srcpkgs/retroarch/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'retroarch'
 pkgname=retroarch
 version=1.9.0
-revision=2
+revision=3
```

Comment:
This doesn't need a revbump; the change that introduced the typo didn't rebuild the package, so the `armv7l` package still includes neon support.